build/mongodl.py

```diff
@@ -214,9 +222,21 @@ def _import_json_data(db, json_file):
                 )
 
 
+def _mkdir(dirpath):
```

Comment:
Was `parents=True` among the unsupported Python features? If so, I would like that to be documented here (reason for deliberately not using an otherwise preferable feature).